lib/matplotlib/widgets.py

```diff
@@ -2699,10 +2699,6 @@ def _press(self, event):
             self.update()
 
         if self._active_handle is None:
-            x = event.xdata
-            y = event.ydata
-            self.visible = False
-            self.extents = x, x, y, y
```

Comment:
Too much! Lines 2702, 2703, and 2705 should be restored. They act to reset the extent of the selector on every click, even if you don't drag it. Otherwise the previous extent persists.

I thought I had left line 2704 alone?